rest-api/core-rest/src/test/java/uk/gov/gchq/gaffer/rest/service/v1/OperationServiceV1IT.java

```diff
@@ -19,10 +19,6 @@
 import uk.gov.gchq.gaffer.rest.RestApiTestClient;
 import uk.gov.gchq.gaffer.rest.service.impl.OperationServiceIT;
 
-/*
- * TODO: look in to why this fails when run other integration tests but passes
- *       when run in isolation.
- */
```

Comment:
I know about this one. This was a problem when I was working on the tests before. It's been documented in #2702 so this is not going to be forgotten once this comment is removed.

Reply:
Okay good to know, I couldn't recreate but good it has its own issue